models/src/main/scala/coop/rchain/models/BlockMetadata.scala

```diff
@@ -70,6 +73,8 @@ object BlockMetadata {
       weightMap(b.body.state),
       b.body.state.blockNumber,
       b.seqNum,
-      invalid
+      invalid,
+      // this value is not used anywhere down the call pipeline, so its safe to set it to false
+      isDirectlyFinalized = false
```

Comment:
We should try to remove this function in the future.

For now maybe better to include directly finalized flag as a parameter, the same as invalid, to be more clear.

Reply:
Im not sure - do you want to change something here or you are ok?

Reply:
Change parameters to include directly finalized flag. So the caller should be aware that finalized info is not read from block itself.